cmake/modules/extensions.cmake

```diff
@@ -4802,6 +4802,14 @@ endmacro()
 # loadable extensions (llexts).
 #
 
+# Usage:
+#   add_llext_target(<target_name>
+#                    OUTPUT  <output_file>
+#                    SOURCES <source_file>
+#                    [C_FLAGS <flags> ...]
```

Comment:
let's have a dedicate function for `C_FLAGS`, perhaps `llext_compile_options(<target> <options>)`